hutch_python/tests/conftest.py

```diff
@@ -246,3 +247,40 @@ def recv_json(self, *args, **kwargs):
 def dummy_zmq_lcls2(monkeypatch):
     # monkeypatch such that the lcls2 daq doesn't actually connect to things
     monkeypatch.setattr(zmq, 'Context', DummyZMQContext)
+
+
+this_test_ophydobj = []
+
+
+@pytest.fixture(scope='session', autouse=True)
+def register_ophydobj():
+    """
+    Save references to all the ophydobj we create.
+    """
+    OphydObject.add_instantiation_callback(this_test_ophydobj.append)
+
+
+@pytest.fixture(scope='function', autouse=True)
+def cleanup_ophydobj():
```

Comment:
These would be great candidates for something like `pcdsutils.tests.ophyd_helpers`

Reply:
Added an issue